iosTraining-Reeen/APICaller.swift

```diff
@@ -10,23 +10,29 @@ import YumemiWeather
 
 protocol WeatherServiceProtocol: AnyObject {
     var delegate: WeatherServiceDelegate? { get set }
-
+    
     func getWeatherInformation()
 }
 
 protocol WeatherServiceDelegate: AnyObject {
-    func weatherService(_ weatherService: WeatherServiceProtocol, didUpdateCondition weatherInfo: String)
+    func weatherService(_ weatherService: WeatherServiceProtocol, didUpdateCondition weatherData: WeatherData)
     func weatherService(_ weatherService: WeatherService, didFailWithError error: Error)
 }
 
 final class WeatherService: WeatherServiceProtocol {
     
+    private let decoder = JSONDecoder()
     weak var delegate: WeatherServiceDelegate?
     
     func getWeatherInformation() {
         do {
-            let weatherInfo = try YumemiWeather.fetchWeatherCondition(at: "tokyo")
-            delegate?.weatherService(self, didUpdateCondition: weatherInfo)
+            let jsonString = "{\"area\": \"tokyo\", \"date\": \"2020-04-01T12:00:00+09:00\"}"
```

Comment:
[FYI]
Extended string delimiters を使うとエスケープが不要になります。

```suggestion
            let jsonString = #"{"area": "tokyo", "date": "2020-04-01T12:00:00+09:00"}"#
```

参考: [Swift の文字列リテラルにおける特殊文字のエスケープ - Qiita](https://qiita.com/daichidaiji/items/9b1179ae32d160292826)

Reply:
こちらで修正しました。
https://github.com/yumemi-inc/iosTraining-Reeen/pull/6/commits/c3129f1aee66a20076cf4b648f7ed7a3ebacd607